### PR TITLE
Deduplicated messages sensors: LOGBROKER-8733

### PR DESCRIPTION
**Pull request by FloatingCrowbar**

### CODE REVIEW

File: ydb/core/persqueue/partition.h [hunk 1]
```diff
@@ -756,6 +756,9 @@ class TPartition : public TActorBootstrapped<TPartition> {
     NKikimr::NPQ::TMultiCounter MsgsWrittenTotal;
     NKikimr::NPQ::TMultiCounter MsgsWrittenGrpc;;
 
+    NKikimr::NPQ::TMultiCounter MsgsDiscarded;;
```
Comment (alexnick88) [inline ydb/core/persqueue/partition.h:759]: Double ;;.

File: ydb/core/persqueue/ut/counters_ut.cpp [hunk 2]
```diff
@@ -194,6 +197,14 @@ Y_UNIT_TEST(PartitionFirstClass) {
         const TString referenceCounters = NResource::Find(TStringBuf("counters_datastreams.html"));
         UNIT_ASSERT_VALUES_EQUAL(countersStr.Str() + "\n", referenceCounters);
     }
+    // CmdWrite(0, "sourceid2", TestData(), tc, false);
```
Comment (alexnick88) [inline ydb/core/persqueue/ut/counters_ut.cpp:200]: Remove this comments.

**Review by alexnick88 — DISMISSED** [review]

File: ydb/core/persqueue/partition_write.cpp [hunk 3]
```diff
@@ -939,7 +939,9 @@ TPartition::ProcessResult TPartition::ProcessRequest(TWriteMsg& p, ProcessParame
                 );
 
                 TabletCounters.Cumulative()[COUNTER_PQ_WRITE_ALREADY].Increment(1);
+                MsgsDiscarded.Inc();
```
Comment (alexnick88) [inline ydb/core/persqueue/partition_write.cpp:942]: Is it correct to Inc MsgsDiscarded when this is mirrored partition? In this case MsgsDiscarded will be inited with default constructor.

**Review by alexnick88 — DISMISSED** [review]

File: ydb/core/persqueue/partition_write.cpp (hunk 3 above)
Comment (FloatingCrowbar) [reply ydb/core/persqueue/partition_write.cpp:942]: Yes. Inc() of default-constructed multi-counter makes no effect.

**Review by alexnick88 — DISMISSED** [review]

**Review by alexnick88 — APPROVED** [review]

**Pull request merged**